src/packageurl/contrib/url2purl.py

```diff
@@ -37,6 +38,9 @@
     from urllib.parse import urlparse  # Python 3
     from urllib.parse import unquote_plus
 
+from pip._internal.wheel import Wheel
```

Comment:
This is all internal and not API... and the library may not have been installed with pip.
How could you document this dependency? Should this is an extra in setup.py? and what if the API changes? Could this be vendored here instead?

Reply:
@pombredanne I've remove the dependency on pip internal in the latest commit.

Reply:
Thanks!